logins-api/android/LoginsApi/loginsapi/build.gradle

```diff
@@ -22,6 +22,12 @@ android {
         }
     }
 
+    // Help folks debugging by including symbols in our native libraries.  Yes, this makes the
```

Comment:
Generally this type of thing is better as a little Pre: commit.

Reply:
Got it.